src/modules/fw_path_navigation/FixedwingPositionControl.cpp

```diff
@@ -2458,27 +2454,60 @@ FixedwingPositionControl::reset_landing_state()
+	// air density into account.
+
+	// Drag modelling (parasite drag): calculate mapping airspeed-->throttle, assuming a linear relation with different gradient
+	// above and blow trim. This is tunable thorugh FW_ASPD_MIN and FW_ASPD_MAX.
+	float throttle_trim = _param_fw_thr_trim.get(); // throttle required at sea level during standard conditions.
+
+	if (PX4_ISFINITE(airspeed_sp) && _param_fw_thr_aspd_min.get() > FLT_EPSILON
+	    && airspeed_sp < _param_fw_airspd_trim.get()) {
+		throttle_trim = _param_fw_thr_trim.get() - (_param_fw_thr_trim.get() - _param_fw_thr_aspd_min.get()) /
+				(_param_fw_airspd_trim.get() - _param_fw_airspd_min.get()) * (_param_fw_airspd_trim.get() - airspeed_sp);
+
+	} else if (PX4_ISFINITE(airspeed_sp) && _param_fw_thr_aspd_max.get() > FLT_EPSILON
+		   && airspeed_sp > _param_fw_airspd_trim.get()) {
+		throttle_trim = _param_fw_thr_trim.get() + (_param_fw_thr_aspd_max.get() - _param_fw_thr_trim.get()) /
+				(_param_fw_airspd_max.get() - _param_fw_airspd_trim.get()) * (airspeed_sp - _param_fw_airspd_trim.get());
+	}
+
+	// Drag modelling (induced drag): calculate mapping load factor--> throttle, assuming a linear relation that is independent of
+	// airspeed. This is tunable through FW_INDCD_DRG_RTO (ratio of induced drag to total drag at trim conditions).
+	// The assumption further is that FW_THR_TIM is set correctly and reflects the thrust equivalent of the drag at trim, and that
+	// throttle and thrust are linear --> double the load factor requires double trim throttle * FW_INDCD_DRG_RTO.
+
 	float weight_ratio = 1.0f;
 
 	if (_param_weight_base.get() > FLT_EPSILON && _param_weight_gross.get() > FLT_EPSILON) {
 		weight_ratio = math::constrain(_param_weight_gross.get() / _param_weight_base.get(), MIN_WEIGHT_RATIO,
 					       MAX_WEIGHT_RATIO);
 	}
 
+	float load_factor_from_bank_angle = 1.f;
+
+	if (PX4_ISFINITE(_roll)) {
+		load_factor_from_bank_angle = constrain(1.f / cosf(_roll), 1.f, 2.f);
+	}
+
+	const float increased_throttle_from_load_factor = _param_fw_indcd_drg_rto.get() * (load_factor_from_bank_angle - 1.f) *
+			weight_ratio * _param_fw_thr_trim.get();
+
+	throttle_trim += increased_throttle_from_load_factor;
+
+	// scale throttle as a function of sqrt(rho0/rho), which is the same relation as TAS/EAS
 	float air_density_throttle_scale = 1.0f;
 
 	if (PX4_ISFINITE(_air_density)) {
-		// scale throttle as a function of sqrt(rho0/rho)
 		const float eas2tas = sqrtf(CONSTANTS_AIR_DENSITY_SEA_LEVEL_15C / _air_density);
 		const float eas2tas_at_5000m_amsl = sqrtf(CONSTANTS_AIR_DENSITY_SEA_LEVEL_15C / AIR_DENSITY_STANDARD_ATMOS_5000_AMSL);
 		air_density_throttle_scale = constrain(eas2tas, 1.f, eas2tas_at_5000m_amsl);
 	}
 
-	// compensate trim throttle for both weight and air density
-	return math::constrain(throttle_trim * sqrtf(weight_ratio) * air_density_throttle_scale, throttle_min, throttle_max);
+	return math::constrain(throttle_trim * air_density_throttle_scale, throttle_min, throttle_max);
```

Comment:
@sfuhrer We should think about if we also need to scale `throttle_max` with air density.